### PR TITLE
Delete habit + Delete unit test

### PR DESCRIPTION
**Pull request by Chanpech**
The delete habit branch is ready to merge. This branch composes of the delete endpoint and the delete unit test.  One area that might need further reviews is what to return.

### CODE REVIEW

**Review by tjzetty — COMMENTED** [review]
Should probably clean up habit.ts. I am not super familiar with the backend API endpoints but probably shouldn't PR huge comment blocks. Looks really good though from what I can tell.

File: apps/backend/package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
         "jsonwebtoken": "^8.5.1",
         "passport": "^0.5.2",
         "passport-jwt": "^4.0.0",
-        "passport-local": "^1.0.0"
+        "passport-local": "^1.0.0",
+        "supertest": "^6.2.2"
```
Comment (tjquillan) [inline apps/backend/package.json]: Supertest is not needed as a dependency

File: apps/backend/src/routes/habit.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { Router } from "express";
+import { userInfo } from "os";
```
Comment (tjquillan) [inline apps/backend/src/routes/habit.ts]: This seem like it might have been imported by accident?

File: apps/backend/src/routes/habit.ts [hunk 3]
```diff
@@ -64,4 +65,47 @@ router.get(
...
+         * For now I am thinking it might have something to do with findMany() and deleteMany().
+        
+        const habit = await prisma.habit.deleteMany({
+          where:{
+            id: habitId,
+          }
+        });
+      })
+);
+*/
```
Comment (tjquillan) [inline apps/backend/src/routes/habit.ts]: Comment can probably be removed

File: apps/backend/test/habit.test.ts [hunk 4]
```diff
@@ -2,6 +2,7 @@ import { agent, SuperAgentTest } from "supertest";
 import prismaMock from "./prisma-mock";
 import app from "../src/app";
 import { createHash } from "../src/auth/hash";
+import { prisma } from "@prisma/client";
```
Comment (tjquillan) [inline apps/backend/test/habit.test.ts]: Also seems to be an accidental import?

**Review by tjquillan — CHANGES_REQUESTED** [review]

**Review by tjquillan — APPROVED** [review]

**Pull request merged**